server/game/core/utils/Contract.ts

```diff
@@ -55,6 +55,12 @@ export function assertNotNullLike<T>(val: T, message?: string): asserts val is N
     }
 }
 
+export function assertIsNullLike<T>(val: T, message?: string): asserts val is null | undefined {
+    if (val != null) {
```

Comment:
This check catches both null and undefined?

Reply:
Yeah, this is like the one good use case for == instead of ===